mod_ci/controllers.py

```diff
@@ -564,6 +564,17 @@ def start_ci():
                 except KeyError:
                     g.log.critical("Didn't find a SHA value for a newly opened PR!")
                     g.log.debug(payload)
+                # Check if user blacklisted
+                if BlockedUsers.query.filter(BlockedUsers.userID == payload[
+                    'pull_request']['user']['id']).first() is not None:
+                    log.critical("User Blacklisted")
+                    gh_commit.post(
+                        state=Status.ERROR,
+                        description="CI start aborted. You may be blocked from accesing this functionality",
```

Comment:
Typo - "accessing".

Also maybe change it to something short like "Tests aborted - user is blacklisted.". What'd you think @canihavesomecoffee ?

Reply:
This is the message posted to the user I think so this should do the job, will fix the typo. Thanks!